README.md

```diff
@@ -89,14 +113,18 @@ var app = express();
 
 Describe the differences between Express and Rails as backend frameworks.
 
-```text
+```Express is lightweight and fast while Rails is resourceful but slower.
+
+Rails require migration to change Schema while express schema can be updated on the fly.
```

Comment:
👍 also, it is mongo that provides the flexible schema, not just express, as you can use a SQL db w/ express